### ISSUE TITLE
Modal refactoring followup

### ISSUE THREAD

**Issue by marekrjpolak** [trezor/trezor-suite#5205]
(Mostly) small improvements needed after #5132:

- wrong `text-align` of modal titles, as seen [in this snapshot](https://github.com/trezor/trezor-suite/blob/07049a55547e1fbb1005485c577da0a768f40c7a/packages/integration-tests/projects/suite-web/snapshots/wallet/cardano.test.ts/cardano-advanced-settings.snap.png)
- nicer standalone modals ([bridge](https://suite.trezor.io/web/bridge), [version](https://suite.trezor.io/web/version), [firmware](https://suite.trezor.io/web/firmware) etc.)
- fix Confirm on device modals (title below image, no title bar => no close/guide buttons) [like this](https://www.notion.so/satoshilabs/Device-modal-improvements-c370e7c20fb541a9bdf67b722a973e6f
)

<sub>This list is incomplete, you can help by expanding it :)</sub>

**Comment by matejkriz** [comment]
- wrong alignment fixed
- nicer standalone modals is not that much followup, more like a separate issue, let's wait for proper specification
- another issue created for "Confirm on device modals" https://github.com/trezor/trezor-suite/issues/5237

**Comment by bosomt** [comment]
**QA OK**

**Info:**
 - Suite version: web 22.5.0 (7281ac61483e38d974625c2505bfe5efd519aacb)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:99.0) Gecko/20100101 Firefox/99.0
 - OS: MacIntel
 - Screen: 1680x1050
 - Device: 
 - Transport: N/A
